engines/python/setup/djl_python/utils.py

```diff
@@ -51,29 +51,25 @@ def parse_input_with_formatter(
         try:
             content_type = item.get_property("Content-Type")
             input_map = decode(item, content_type)
+            _inputs, _param, is_client_side_batch[i] = _parse_inputs_params(
```

Comment:
Why `_` on local variables? I'm not aware of this as a Python convention. (Note: I recognize it was like that before this PR).

Reply:
there are also [places](https://github.com/deepjavalibrary/djl-serving/blob/master/engines/python/setup/djl_python/utils.py#L89-L94) in this file where that convention is followed. I don't know why. I do not want to mix those changes in this PR, will raise a separate PR to isolate the changes.